Tools/PowershellModule/doc/new-private-repository-azure.md

```diff
@@ -556,15 +529,12 @@ For more information on Azure Functions, visit their Docs article: [Introduction
         - Value: [Value]
         - Deployment slot setting: Disabled
 
-#### Import Windows Package Manager API
-
-**How to:**
+### Import Windows Package Manager API
 
 1. Open a PowerShell window on your computer.
 
     ``` PowerShell
     PS C:\> $ArchiveFunctionZip = "C:\Projects\winget-cli-restsource\src\WinGet.RestSource.Infrastructure\CompiledFunctions.zip"
```

Comment:
Not a fan of the name archive functions.

The path and name of the .zip is out of date.
Point to releases, not non-existent paths in source